### PR TITLE
[PC-1069] Portenta Cat m1 shield Datasheet Fix

### PR DESCRIPTION
**Pull request by Jorgetrujilloroman**
## What This PR Changes
- [Jira task](https://arduino.atlassian.net/browse/PC-1069)
- Fix the certifications section of the Datasheet regarding the "Declaration of Conformity to EU RoHS & REACH 211 01/19/2021" and other grammatical corrections.

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/04.pro/shields/portenta-cat-m1-nb-iot-gnss-shield/datasheet/datasheet.md [hunk 1]
```diff
@@ -60,10 +60,10 @@ Enter the world of global asset tracking with the **Arduino® Portenta Cat. M1/N
...
 
 - **Remote node monitoring**
-  The **Arduino® Portenta Cat. M1/NB IoT GNSS Shield** can relay geo-tagged data from local sensors located worldwide to provide real time insight for increasing your business revenues. 
+  The **Arduino® Portenta Cat. M1/NB IoT GNSS Shield** can relay geo-tagged data from local sensors located worldwide to provide real-time insight for increasing your business revenues. 
 
 - **Fleet management**
-  Manage your MaaS (Mobility as a Service) solution across the city or between borders. Track, analyse and dynamically manage your fleet to optimise fuel usage, increase your customers satisfaction and reduce transport times. Enable predictive maintenance and remote diagnostics to ensure your business runs smoothly with minimal downtime.
+  Manage your MaaS (Mobility as a Service) solution across the city or between borders. Track, analyse and dynamically manage your fleet to optimize fuel usage, increase your customers satisfaction and reduce transport times. Enable predictive maintenance and remote diagnostics to ensure your business runs smoothly with minimal downtime.
```
Comment (jcarolinares) [inline content/hardware/04.pro/shields/portenta-cat-m1-nb-iot-gnss-shield/datasheet/datasheet.md:66]: USA version, well done, we are writing USA english

File: content/hardware/04.pro/shields/portenta-cat-m1-nb-iot-gnss-shield/datasheet/datasheet.md [hunk 2]
```diff
@@ -155,16 +155,17 @@ Now that you have gone through the basics of what you can do with the board, you
...
 We declare under our sole responsibility that the products above are in conformity with the
 essential requirements of the following EU Directives and therefore qualify for free movement
 within markets comprising the European Union (EU) and European Economic Area (EEA).
 Radio Equipment Directive (RED) 2014/53/EU
 
-We declare under our sole responsibility that the products above are in conformity with the
-essential requirements of the following EU Directives and therefore qualify for free movement
-within markets comprising the European Union (EU) and European Economic Area (EEA).
-Radio Equipment Directive (RED) 2014/53/EU
+### Declaration of Conformity to EU RoHS & REACH 211 01/19/2021
```
Comment (jcarolinares) [inline content/hardware/04.pro/shields/portenta-cat-m1-nb-iot-gnss-shield/datasheet/datasheet.md:165]: remove this change

If I'm not wront you are going to duplicate a text

**Review by jcarolinares — CHANGES_REQUESTED** [review]
Reviewed

@Jorgetrujilloroman perform the changes requested

File: content/hardware/04.pro/shields/portenta-cat-m1-nb-iot-gnss-shield/datasheet/datasheet.md (hunk 2 above)
Comment (Jorgetrujilloroman) [reply content/hardware/04.pro/shields/portenta-cat-m1-nb-iot-gnss-shield/datasheet/datasheet.md:165]: The text is duplicated in the actual Datasheet. I am actually removing the duplicated text and adding the necessary information.

image

Comment (jcarolinares) [reply content/hardware/04.pro/shields/portenta-cat-m1-nb-iot-gnss-shield/datasheet/datasheet.md:165]: I see! then proceed!

**Review by jcarolinares — APPROVED** [review]
Looks good to me

Please @Jorgetrujilloroman merge!

**Review by jcarolinares — APPROVED** [review]

**Comment by Jorgetrujilloroman** [comment]
Merging to Main

**Pull request merged**